README.md

```diff
@@ -51,6 +51,7 @@ source:
     #   username: "USERNAME"
     #   password: "PASSWORD"
 target:
+  repoName: "myrepo"
```

Comment:
I would document what's this setting for (you can do this later on since other settings are not documented either)

Reply:
yes, I am working on the documentation in a different PR.